src/index.html

```diff
@@ -1,16 +1,48 @@
 <!doctype html>
 <html lang="en">
+
   <head>
```

Comment:
Remove blank lines between ALL parent and child tags
```suggestion
<html lang="en">
  <head>
```